dspace-api/src/main/java/org/dspace/discovery/SolrServiceSearchPlugin.java

```diff
@@ -19,5 +19,6 @@
  */
 public interface SolrServiceSearchPlugin {
 
-    public void additionalSearchParameters(Context context, DiscoverQuery discoveryQuery, SolrQuery solrQuery);
+    public void additionalSearchParameters(Context context, DiscoverQuery discoveryQuery, SolrQuery solrQuery)
```

Comment:
Could we add quick JavaDocs to this method while we are updating it?

Reply:
It's not specific to this PR, but we can add JavaDocs